code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -995,3 +997,89 @@
 	hat = new_hat
 	new_hat.forceMove(src)
 	update_icons()
+
+/mob/living/silicon/robot/MouseDrop_T(mob/living/M, mob/living/user)
+	. = ..()
+	if(!(M in buckled_mobs))
+		buckle_mob(M)
+
+/mob/living/silicon/robot/buckle_mob(mob/living/M, force = 0, check_loc = 1)
```

Comment:
USE THE TRUE FALSE DEFINES HOW MANY FUCKING TIMES DO I HAVE TO FUCKING SAY IT AHHHHHHHHH